go.mod

```diff
@@ -1,8 +1,8 @@
 module stackerbuild.io/stacker
 
-go 1.21.0
+go 1.22
 
-toolchain go1.21.6
+toolchain go1.22.6
```

Comment:
I think you need 1.22.8 for the x/net@v0.33.0 to fix the net cve

Reply:
ah ok, I will update my tools and redo the go mod tidy, thanks